src/js/common/schemaform/ApplicationStatus.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { connect } from 'react-redux';
+
+import { formLinks } from '../../user-profile/helpers';
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+import ProgressButton from '../../common/components/form-elements/ProgressButton';
+import Modal from '../../common/components/Modal';
+import { removeInProgressForm } from '../../common/schemaform/sip-api';
+
+export class ApplicationStatus extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modalOpen: false
+    };
+  }
+
+  removeForm = () => {
+    this.toggleModal();
+    removeInProgressForm(this.props.formId).then(() => {
+      window.location.href = formLinks[this.props.formId];
+    });
+  }
+
+  toggleModal = () => {
+    this.setState({ modalOpen: !this.state.modalOpen });
+  }
+
+  render() {
+    const { formId, profile, login, applyText, titleText, showApplyButton } = this.props;
+
+    if (login.currentlyLoggedIn && !profile.loading && profile.savedForms.some(({ form }) => form === formId)) {
+      const formData = profile.savedForms.find(({ form }) => form === formId);
+      const { last_updated: lastSaved, expires_at: expirationTime } = formData.metadata;
+      const expirationDate = moment.unix(expirationTime).format('M/D/YYYY');
+      const isExpired = moment(expirationDate).isBefore();
+
+      if (!isExpired) {
+        const lastSavedDateTime = moment.unix(lastSaved).format('M/D/YYYY [at] h:mm a');
+
+        return (
+          <div className="usa-alert usa-alert-info no-background-image sip-application-status">
```

Comment:
This is all based on the FormItem component, but it was different enough that it didn't seem like I could reuse it. But might be worth figure out if we can make some of the date logic shared.